### PR TITLE
[NEXTGEN] Detekt is good for code

### PR DESCRIPTION
**Pull request by gabrielvicenteYT**
This should be quite good when finished.

### CODE REVIEW

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/ModuleMurderMystery.kt [hunk 1]
```diff
@@ -150,8 +150,8 @@ object ModuleMurderMystery : Module("MurderMystery", Category.RENDER) {
 
         val contractLine = IntArray(128 * 7)
 
-        (0 until 7).forEach { y ->
-            (0..128).forEach { x ->
+        for (i in 0..7) { y ->
```
Comment (CzechHek) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/ModuleMurderMystery.kt:153]: 0 until 7 != 0..7
0 until 7 == 0.<7 == 0..6

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/ModuleMurderMystery.kt [hunk 2]
```diff
@@ -169,7 +169,7 @@ object ModuleMurderMystery : Module("MurderMystery", Category.RENDER) {
         var lastNonEmptyScanline = -1
         var emptyScanlines = 0
 
-        (0 until 128).forEach { x ->
```
Comment (CzechHek) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/ModuleMurderMystery.kt:172]: 0 until 128 != 0..128

**Comment by CzechHek** [comment]
these loops should be simplified to repeat(40) {} and so on

note: repeat(41) for 0..40 cause 0-40 is 41 iterations, repeat starts from 0 and goes to 40 with repeat(41)

**Comment by gabrielvicenteYT** [comment]
> these loops should be simplified to repeat(40) {} and so on
> 
> note: repeat(41) for 0..40 cause 0-40 is 41 iterations, repeat starts from 0 and goes to 40 with repeat(41)

I'd first have to check if it has any performance cost.

**Comment by CzechHek** [comment]
> I'd first have to check if it has any performance cost.

40 iterations clearly has a huge performance impact yes and the difference between for i in 0..40 and repeat(41) is also huge so huge in fact that for i in 0..40 temporarily creates an IntRange and therefore is worse 😳

**Comment by gabrielvicenteYT** [comment]
> > I'd first have to check if it has any performance cost.
> 
> 40 iterations clearly has a huge performance impact yes and the difference between for i in 0..40 and repeat(41) is also huge so huge in fact that for i in 0..40 temporarily creates an IntRange and therefore is worse 😳

You're right, thanks!

**Comment by ghost** [comment]
> > these loops should be simplified to repeat(40) {} and so on
> > note: repeat(41) for 0..40 cause 0-40 is 41 iterations, repeat starts from 0 and goes to 40 with repeat(41)
> 
> I'd first have to check if it has any performance cost.

JustinTimbelakeStareGIF

**Pull request closed without merging**